packages/react-storage/src/components/StorageBrowser/context/elements/__tests__/defaults.spec.tsx

```diff
@@ -1,9 +1,34 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import { elementsDefault } from '../defaults';
 import { ButtonElementVariant } from '../definitions';
+import { createStorageBrowser } from '../../../createStorageBrowser';
 
 describe('elementsDefault', () => {
+  it('should render the elementsDefault when used', async () => {
+    const listLocations = jest.fn(() =>
+      Promise.resolve({ locations: [], nextToken: undefined })
+    );
+    const config = {
+      getLocationCredentials: jest.fn(),
+      listLocations,
+      region: 'region',
+      registerAuthListener: jest.fn(),
+    };
+    const { StorageBrowser } = createStorageBrowser({
+      elements: elementsDefault,
+      config,
+    });
+
+    render(<StorageBrowser />);
+    await waitFor(() => {
```

Comment:
Are we sure the tests inside `waitFor` are actually passing or just that they are not throwing?

Reply:
They are passing. Should I use `findBy*` instead?

Reply:
If the assertions fail inside `waitFor` the test will still pass. Ideally should be able to wrap the render call inside `waitFor` and run the assertions outside